apps/roam/src/utils/fetchEmbeddingsForNodes.ts.ts

```diff
@@ -0,0 +1,69 @@
+import { DiscourseGraphContent } from "./types";
```

Comment:
remove the additional `.ts` in the filename

`apps/roam/src/utils/fetchEmbeddingsForNodes.ts.ts`